sage-ios/SAGE/SAGE/AddAnnouncementTableViewController.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  AddAnnouncementTableViewController.swift
+//  SAGE
+//
+//  Created by Sameera Vemulapalli on 11/24/15.
+//  Copyright © 2015 Cal Blueprint. All rights reserved.
+//
+
+import UIKit
+
+class AddAnnouncementTableViewController: UITableViewController {
```

Comment:
can you rename this to something like SelectAnnouncemtnSchoolTableViewController? sounds too similar to AddAnnouncementVC